### PR TITLE
Add support for OpenMRS 3.x and update README

### PR DESCRIPTION
**Pull request by donaldkibet**
### What does this PR do ?

- Add some  rest endpoint documentation for resources available
- Update README with additional information 
- Add some additional end-point, `get appointments by date and status`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,346 @@
...
+
+# Setup instructions
+
+This module is incompatiable with the default openmrs-appointment-scheduling module, please ensure you remove it either from the admin-page or running this command below from where your modules are located in tomcat directory i.e `/var/lib/OpenMRS/modules` on linux system
+
+```
+sudo rm appointmentscheduling-1.13.0.omod
+```
+
+If you get database error while running the module, remove this tables below. by running the following sql statements.
```
Comment (mks-d) [inline README.md:20]: >remove this tables below

this → these tables

**Review by mks-d — COMMENTED** [review]
Thanks a lot for expanding the README, that's the way to go 👍 

I didn't read everything yet but I found a small typo.

**Comment by mseaton** [comment]
@donaldkibet / @ojwanganto / @CynthiaKamau - should this old PR be closed in favor of the new PRs you are creating?

**Comment by mseaton** [comment]
> We can close it

I don't seem to have a button to do that.  @donaldkibet ?

**Comment by donaldkibet** [comment]
Closing this PR @mseaton

**Pull request closed without merging**